code/obj/item/device/chameleon.dm

```diff
@@ -122,6 +122,8 @@
 			if (user && ismob(user))
 				user.show_text("You are too far away to do that.", "red")
 			return
+		if (istype(target, /obj/ability_button) || istype(target, /obj/item/grab)) //just don't scan ability buttons or grabs
```

Comment:
...is there not a var or helper proc to check instead?

Reply:
There is one for hud stuff! Just not sure if there is one for grabs.